src/pages/account/AdminPage.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect } from "react";
+import { Link, useParams } from "react-router-dom";
+import styled from "styled-components";
+
+import axiosApi from "../../utils/AxiosApi";
+import Card from "../../components/Card";
+import NavBar from "../../components/NavBar";
+import Button from "../../components/Button";
+
+const AdminPage = () => {
+  const [stories, setStories] = useState([]);
+  const [child, setChild] = useState({});
+  const [id, setId] = useState("");
+  const { childId } = useParams();
+
+  useEffect(() => {
+    const childData = async () => {
+      const response = await axiosApi(`/getChild/${childId}`);
+      const childData = await response.data;
+      const storyData = await response.data.stories;
+      setChild(childData);
+      setStories(storyData);
+    };
+    return childData();
+  }, [childId]);
+
+  const deleteStory = async (event) => {
+    event.preventDefault();
+    if (id) {
+      await axiosApi.post(`deleteStory/${id}`, { childId });
+      const response = await axiosApi(`/getChild/${childId}`);
+      const storyData = await response.data.stories;
+      setStories(storyData);
+    }
+  };
+
+  if (stories.length === 0) {
+    return (
+      <>
+        <NavBar />
+        <p>
+          There are no stories yet. Be the first to write a story and go to your{" "}
+          <Link to="/account">home page</Link>, select {child.firstname} and
+          start writing.
+        </p>
+      </>
+    );
+  }
+
+  return (
+    <>
+      <NavBar />
```

Comment:
Apparently you need `<Navbar>` in all cases. So what can you do to apply the DRY principle here?

Reply:
Only private routes have a navbar. I think I can change it but I will do it when this is merged, else I think I'll get too much merge conflict right now...

Reply:
On line 40 you also have a `<Navbar>` component.
Think about a way where you only need to use that component one for both situations

Reply:
p.s. Don't worry about Merge conflicts. I'm actually glad that you have them. As it's also something you should learn how to resolve them

Reply:
Yes, but I'll fix it when it's merged. Already have merge conflicts right now

Reply:
Fix it here, as this is code that you've added. This is also a good way to learn how to deal with merge conflicts